### PR TITLE
Nav Bar: Indicate Current Page

### LINKED ISSUE #242: [BUG] Nav Bar: Indicate Current Page
## Description

The Nav Bar should have some visual indication (like a blue underline or something) to show the user which page they are currently on.

### PR DESCRIPTION
**Pull request by uma-anand**
## Nav Bar: Indicate Current Page

Issue Number(s): #242 .

What does this PR change and why?

### Checklist

- [ ] Requirements have been implemented
- [ ] Acceptance criteria are met
- [ ] Database schema docs have been updated or are not necessary
- [ ] Code follows design and style guidelines
- [ ] Commits follow guidelines (concise, squashed, etc)
- [x] Relevant reviewers (Senior Dev/EM/Designers) have been assigned to this PR

### Critical Changes

- Database change/migration to run
- Environment config change
- Breaking API change

### Related PRs

- #number_of_pr

### Testing

Enumerate steps to test the functionality of your ticket. This should include edge cases and testing of error handling, if applicable.

### CODE REVIEW

File: web/components/Navbar.tsx [hunk 1]
```diff
@@ -122,6 +123,7 @@ export default function Navbar() {
   const router = useRouter();
   const { user, loading, userData, authorized } = useAuth();
   const role = userData?.role;
+  const [currPage, setCurrPage] = useState("FEED");
```
Comment (katsukixyz) [inline web/components/Navbar.tsx]: You shouldn't have to create new state here - you can get the current route using `router.pathName`

File: web/components/Navbar.tsx [hunk 2]
```diff
@@ -187,20 +189,48 @@ export default function Navbar() {
           spacing={10}
         >
           <Link as={NextLink} href={Pages.FEED}>
-            <Text>Feed</Text>
+            <Text
+              color={currPage === "FEED" ? "#57A0D5" : "black"}
```
Comment (katsukixyz) [inline web/components/Navbar.tsx]: Do something like `router.pathName === Pages.FEED`. Use Chakra's semantic tokens that we have defined whenever you can - I think it's `"text-primary"`

**Review by katsukixyz — CHANGES_REQUESTED** [review]

File: web/components/Navbar.tsx (hunk 2 above)
Comment (katsukixyz) [reply web/components/Navbar.tsx]: Also, just noticed this, but make sure you change the hover underline color to be the same as the text color

**Review by katsukixyz — CHANGES_REQUESTED** [review]
Thanks for the changes, I believe I left an additional comment about the underline:
https://github.com/GTBitsOfGood/angels-among-us/pull/263#discussion_r1382451880

File: web/components/Navbar.tsx (hunk 2 above)
Comment (uma-anand) [reply web/components/Navbar.tsx]: whoops I totally did not see that, will do

File: web/components/Navbar.tsx [hunk 3]
```diff
@@ -186,21 +186,73 @@ export default function Navbar() {
...
           spacing={10}
         >
-          <Link as={NextLink} href={Pages.FEED}>
-            <Text>Feed</Text>
+          <Link
+            as={NextLink}
+            href={Pages.FEED}
+            _hover={{
+              textDecoration: "underline",
+              textDecorationColor: "text-primary",
```
Comment (katsukixyz) [inline web/components/Navbar.tsx]: Text decoration color should depend on the current route as well

**Review by katsukixyz — CHANGES_REQUESTED** [review]

**Review by katsukixyz — APPROVED** [review]
Good work on the changes! LGTM 😄
